### PR TITLE
fix: volume map to default Docker Deskotp file sharing (/Users)

### LINKED ISSUE #188: Update the wrapper scripts to mount individual paths
Fix up the wrapper script to mount individual paths rather than / to account for it

```
Fetching package registry https://github.com/blinklabs-io/cardano-up/archive/refs/heads/main.zip
Installing package cardano-node (= 8.9.1)
docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: mkdir /host/Users: file exists: unknown.
ERROR: run hook script exited with error: exit status 125
```

### PR DESCRIPTION
**Pull request by verbotenj**
fixes #188

### CODE REVIEW

File: packages/cardano-cli/files/cardano-cli.sh.gotmpl [hunk 1]
```diff
@@ -27,13 +27,13 @@ else
...
-# We map the host filesystem and node socket into the container
+# We map the /Users and node socket into the container
 docker run \
 	-ti \
 	--rm \
 	-u $(id -u):$(id -g) \
-	-v /:/host \
-	-w /host$(pwd) \
+	-v /Users:/Users \
+	-w $(pwd) \
```
Comment (agaffney) [inline packages/cardano-cli/files/cardano-cli.sh.gotmpl:36]: We want to keep the `/host` prefix for consistency

File: packages/mithril-client/files/mithril-client.sh.gotmpl [hunk 2]
```diff
@@ -9,7 +9,7 @@ for i in ${!_options[@]}; do
         _found=false
 	# remap absolute paths to /host in the container
 	if [[ ${k} =~ ^/ ]]; then
-		k="/host${k}"
+		k="${k}"
```
Comment (agaffney) [inline packages/mithril-client/files/mithril-client.sh.gotmpl:12]: Keep the `/host` prefix

File: packages/mithril-client/files/mithril-client.sh.gotmpl [hunk 3]
```diff
@@ -44,13 +44,13 @@ done
...
-# We map the host filesystem into the container
+# We map the /Users into the container
 docker run \
 	-ti \
 	--rm \
 	-u $(id -u):$(id -g) \
-	-v /:/host \
-	-w /host$(pwd) \
+	-v /Users:/Users \
+	-w $(pwd) \
```
Comment (agaffney) [inline packages/mithril-client/files/mithril-client.sh.gotmpl:53]: Keep the `/host` prefix

**Review by agaffney — CHANGES_REQUESTED** [review]
This is going to break on Linux. You need to do some OS detection in the script to change the handling for the host paths a bit.

**Review by wolf31o2 — APPROVED** [review]

**Pull request merged**